pkg/store/bucket.go

```diff
@@ -995,23 +1014,33 @@ func newSortedMatchers(matchers []*labels.Matcher) sortedMatchers {
 func (b *blockSeriesClient) ExpandPostings(
 	matchers sortedMatchers,
 	seriesLimiter SeriesLimiter,
+	lazyExpandedPostingEnabled bool,
```

Comment:
can we make those to fields of the blockSeriesClient instead of passing them as parameters?

Reply:
Yeah agree, I can make the change.